### PR TITLE
Cv evulnerabilities chnages

### PR DESCRIPTION
**Pull request by v-shazilms**

### CODE REVIEW

File: Samples/BetaSamples/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj [hunk 1]
```diff
@@ -82,17 +82,9 @@
...
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-
-  <Target
-    Name="BuiltProjectOutputGroupDependencies"
-    DependsOnTargets="$(BuiltProjectOutputGroupDependenciesDependsOn)"
-    Returns="@(BuiltProjectOutputGroupDependency)">
+  <Target Name="BuiltProjectOutputGroupDependencies" DependsOnTargets="$(BuiltProjectOutputGroupDependenciesDependsOn)" Returns="@(BuiltProjectOutputGroupDependency)">
     <ItemGroup>
-      <BuiltProjectOutputGroupDependency Include="@(ReferenceCopyLocalPaths->'%(FullPath)');
```
Comment (samarthasthana) [inline Samples/BetaSamples/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj:91]: If there is no change here, please revert the change, prefer the earlier formatting

File: Samples/BetaSamples/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj [hunk 2]
```diff
@@ -82,17 +82,9 @@
     </Content>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-
```
Comment (samarthasthana) [inline Samples/BetaSamples/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj:85]: Same as below, revert if no change

File: Samples/Common/Sample.Common.Beta/Sample.Common.Beta.csproj [hunk 3]
```diff
@@ -10,23 +10,27 @@
...
+    <PackageReference Include="Microsoft.IdentityModel.Abstractions" Version="8.0.1" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net472' ">
     <Reference Include="System" />
     <Reference Include="System.Data" />
     <Reference Include="System.Net.Http" />
   </ItemGroup>
+
+  <ItemGroup>
```
Comment (samarthasthana) [inline Samples/Common/Sample.Common.Beta/Sample.Common.Beta.csproj:33]: Is this part of the CVE fix? Here and below.

File: Samples/V1.0DeltaRosterSample/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj [hunk 4]
```diff
@@ -83,17 +83,9 @@
     </Content>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-
```
Comment (samarthasthana) [inline Samples/V1.0DeltaRosterSample/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj:86]: Same as above

File: Samples/V1.0DeltaRosterSample/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj [hunk 5]
```diff
@@ -83,17 +83,9 @@
...
-  <Target
-    Name="BuiltProjectOutputGroupDependencies"
-    DependsOnTargets="$(BuiltProjectOutputGroupDependenciesDependsOn)"
-    Returns="@(BuiltProjectOutputGroupDependency)">
+  <Target Name="BuiltProjectOutputGroupDependencies" DependsOnTargets="$(BuiltProjectOutputGroupDependenciesDependsOn)" Returns="@(BuiltProjectOutputGroupDependency)">
     <ItemGroup>
-      <BuiltProjectOutputGroupDependency Include="@(ReferenceCopyLocalPaths->'%(FullPath)');
```
Comment (samarthasthana) [inline Samples/V1.0DeltaRosterSample/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj:92]: Same as above

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj [hunk 6]
```diff
@@ -75,17 +75,9 @@
     </Content>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-
-  <Target
```
Comment (samarthasthana) [inline Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj:79]: Same as above

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj [hunk 7]
```diff
@@ -75,17 +75,9 @@
...
-    Name="BuiltProjectOutputGroupDependencies"
-    DependsOnTargets="$(BuiltProjectOutputGroupDependenciesDependsOn)"
-    Returns="@(BuiltProjectOutputGroupDependency)">
+  <Target Name="BuiltProjectOutputGroupDependencies" DependsOnTargets="$(BuiltProjectOutputGroupDependenciesDependsOn)" Returns="@(BuiltProjectOutputGroupDependency)">
     <ItemGroup>
-      <BuiltProjectOutputGroupDependency Include="@(ReferenceCopyLocalPaths->'%(FullPath)');
```
Comment (samarthasthana) [inline Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj:84]: Same as above

File: Samples/BetaSamples/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj (hunk 2 above)
Comment (BalajiUmmadisetti) [reply Samples/BetaSamples/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj]: reverted

File: Samples/Common/Sample.Common.Beta/Sample.Common.Beta.csproj (hunk 3 above)
Comment (BalajiUmmadisetti) [reply Samples/Common/Sample.Common.Beta/Sample.Common.Beta.csproj]: mistakenly added , reverted

File: Samples/V1.0DeltaRosterSample/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj (hunk 4 above)
Comment (BalajiUmmadisetti) [reply Samples/V1.0DeltaRosterSample/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj]: reverted

File: Samples/V1.0DeltaRosterSample/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj (hunk 5 above)
Comment (BalajiUmmadisetti) [reply Samples/V1.0DeltaRosterSample/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj]: reverted

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj (hunk 6 above)
Comment (BalajiUmmadisetti) [reply Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj]: reverted

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj (hunk 7 above)
Comment (BalajiUmmadisetti) [reply Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj]: reverted

**Review by samarthasthana — APPROVED** [review]
